### ISSUE TITLE
Wrong named envvars

### ISSUE THREAD

**Issue by mmattel** [owncloud/ocis#7292]
Just identified when looking into [global envvars](https://doc.owncloud.com/ocis/next/deployment/services/env-vars-special-scope.html) that should start with `OCIS_`

`STORAGE_USERS_PERMISSION_ENDPOINT` --> `OCIS_STORAGE_USERS_PERMISSION_ENDPOINT`

`STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY` --> `OCIS_STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY`

In the **graph** service, same as in userlog, there is the key-pair:
`OCIS_MACHINE_AUTH_API_KEY`
`USERLOG_MACHINE_AUTH_API_KEY` --> `GRAPH_MACHINE_AUTH_API_KEY`

`WEB_UI_CONFIG_FILE` is double present, can be seen in the web service too, we need to check why.

Following envvars are used in both the `userlog` and `clientlog` service. Those should be then either a global one starting with `OCIS_` or the clientlog version adapted to `CLIENTLOG_`:

`USERLOG_DEBUG_ADDR`
`USERLOG_DEBUG_PPROF`
`USERLOG_DEBUG_TOKEN`
`USERLOG_DEBUG_ZPAGES`
`USERLOG_LOG_COLOR`
`USERLOG_LOG_FILE`
`USERLOG_LOG_LEVEL`
`USERLOG_LOG_PRETTY`

@kobergj @dragonchaser

**Comment by kobergj** [comment]
`STORAGE_USERS_PERMISSION_ENDPOINT` and `STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY` are only present in storage-users service and therefore need no global `OCIS` prefix

All `MACHINE_AUTH_API_KEY` envvars can be deleted as they are no longer used because of service accounts

`USERLOG` envvars in clientlog service need of course be changed to `CLIENTLOG...`. That was a copy/paste mistake on implementation and a lazy no-look during review

**Comment by rhafer** [comment]
> All `MACHINE_AUTH_API_KEY` envvars can be deleted as they are no longer used because of service accounts

The `proxy` service is still using machine auth to be able to mint reva tokens for oidc user (and signedurl IIRC). Service accounts won't help to get rid of those cases I think. So we still need `MACHINE_AUTH_API_KEY`

**Comment by kobergj** [comment]
:+1:  let me rephrase

For all services except `proxy` the `MACHINE_AUTH_API_KEY` envvar can be deleted as it is no longer used because of service accounts.

**Comment by kobergj** [comment]
Services `idp` and `frontend` also still need the `MACHINE_AUTH_API_KEY` otherwise tests fail. We need to find out why and replace usage of `machine` authentication with `service` authentication. I'll create seperate tickets for that.
